packages/zipkin-instrumentation-axiosjs/test/integrationTest.js

```diff
@@ -22,53 +23,41 @@ describe('axios instrumentation - integration test', () => {
       done();
     });
   });
+
   after(() => {
     apiServer.close();
   });
-  const getClient = () => wrapAxios(axios, {tracer, serviceName, remoteServiceName});
+
   beforeEach(() => {
     record = sinon.spy();
-    const recorder = {record};
     const ctxImpl = new ExplicitContext();
-    tracer = new Tracer({recorder, ctxImpl});
+    const recorder = {record};
+    tracer = new Tracer({ctxImpl, recorder});
   });
+
+  const getClient = () => wrapAxios(axios, {tracer, serviceName, remoteServiceName});
+
   it('should add headers to requests', done => {
     tracer.scoped(() => {
       const zipkinAxiosClient = getClient();
       const urlPath = '/weather/wuhan';
       const url = `http://${apiHost}:${apiPort}${urlPath}?index=10&count=300`;
-      zipkinAxiosClient
```

Comment:
@openzipkin/zipkin-js-champions in I think every integration test, we copy/pasted the same code which didn't actually test that headers were added!